lib/plug/redoc_ui.ex

```diff
@@ -60,4 +106,19 @@ defmodule Redoc.Plug.RedocUI do
     |> put_resp_content_type("text/html")
     |> send_resp(200, EEx.eval_string(@index_html, opts))
   end
+
+  defp encode_options(opts) do
+    Enum.reduce(opts, [], fn {key, value}, acc ->
+      case Enum.member?(@redoc_options, key) do
+        true -> [{to_kebab_case(key), value} | acc]
+        false -> acc
+      end
+    end)
```

Comment:
This could be done by using `Keyword.take/2`:

```elixir
Keyword.take(opts, @redoc_options)
|> Enum.map(fn {key, value} -> {to_kebab_case(key), value} end)

Reply:
Oh, that's much better!